bin/cml/pr.js

```diff
@@ -39,6 +39,24 @@ exports.builder = (yargs) =>
         conflicts: ['merge', 'rebase'],
         description: 'Try to squash-merge the pull request upon creation.'
       },
+      branch: {
+        type: 'string',
+        description: 'Set a custom branch name for pull request.'
+      },
+      title: {
+        type: 'string',
+        description: 'Title of the created pull request.'
+      },
+      body: {
+        type: 'string',
+        description:
+          'Description/body of created pull request [string or file]. Ex "My cml pr", "reports/experiment-results.md"'
```

Comment:
```suggestion
          'Body for the created pull request [string or file].'
```

Not a fan of verbose examples in help strings. 😿

Reply:
also not files!